cmake/hipsycl-config.cmake.in

```diff
@@ -82,17 +82,17 @@ unset(HIPSYCL_NUM_AVAILABLE_BACKENDS)
 unset(HIPSYCL_DEFAULT_PLATFORM)
 
 set(HIPSYCL_SYCLCC_EXTRA_ARGS "--hipsycl-platform=${HIPSYCL_PLATFORM}" CACHE STRING
-        "Extra compiler flags specific to syclcc that are incompatible with GCC/Clang")
+        "Extra compiler flags specific to syclcc that are incompatible with GCC/Clang" FORCE)
```

Comment:
Thanks for looking into it!

I think we should simply turn `HIPSYCL_SYCLCC_EXTRA_ARGS` into a regular, non-cache variable, then there's no need to `FORCE` anything later. That still works on my machine.

I remember having something like `CMAKE_CXX_FLAGS` in mind that can be set from command line in case the user wants to provide extra syclcc-specific arguments, but since that variable is overwritten in the script anyway, that never worked.

Reply:
Thanks for the review, good point :)